Sming/Makefile

```diff
@@ -319,6 +320,9 @@ docs/wiki/Home.md:
 	$(Q) $(GIT) submodule update --init ../docs/wiki	
 
 docs/api/error.log:
+ifndef DOXYGEN
+	$(error doxygen not found - not building API docs)
+endif
```

Comment:
I thought that too but the `$(error...)` command throws a make error, hence terminating make.